WordPress/Classes/ViewRelated/Blog/DiscussionSettingsViewController.swift

```diff
@@ -66,6 +65,18 @@ public class DiscussionSettingsViewController : UITableViewController
     
 
     // MARK: - Persistance!
+    private func refreshSettings() {
+        let service = BlogService(managedObjectContext: settings.managedObjectContext)
+        service.syncSettingsForBlog(settings.blog,
```

Comment:
Maybe just `blog` instead of `settings.blog`.  The `settings` is just a convenience accessor right?  So using it is sort of like calling `blog.settings.blog`.